internal/minioclient/client.go

```diff
@@ -9,11 +9,13 @@ import (
 	"io"
 
 	minio "github.com/minio/minio-go/v6"
+	"github.com/spacemonkeygo/monkit/v3"
 	"github.com/zeebo/errs"
 )
 
 // MinioError is class for minio errors.
 var MinioError = errs.Class("minio error")
+var mon = monkit.Package()
```

Comment:
this can be folded into a single var, e.g.

```
var (
    // MinioError is class for minio errors.
    MinioError = errs.Class("minio error")

    mon = monkit.Package()
)
```

Reply:
done